src/components/home.tsx

```diff
@@ -0,0 +1,25 @@
+import {Component, FC} from 'react'
+import templatesConfig from '../templates/templates_config'
+import { ListGroup } from 'react-bootstrap';
+
+const Home: FC = () => {
+    
+  const homeFlexItems = Object.keys(templatesConfig).map((key) => (
+    <ListGroup.Item key={key}>
+      <a href={`/app/${key}`}>
+      {templatesConfig[key as keyof typeof templatesConfig].title}
+      </a>
+    </ListGroup.Item>
+  ));
+    
+  return (
+    <div>
+      <h1>Home</h1>
```

Comment:
I wonder if instead of `Home`, we should say something like "Choose a Document Type".